token/core/identity/msp/wf.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package msp
+
+import (
+	"github.com/IBM/idemix/common/flogging"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/common"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/idemix"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/identity/msp/x509"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/driver/config"
+	"github.com/pkg/errors"
+)
+
+var logger = flogging.MustGetLogger("token-sdk.driver.identity.tms")
+
+const (
+	// OwnerMSPID is the default MSP ID for the owner wallet
+	OwnerMSPID = "OwnerMSPID"
+	// IssuerMSPID is the default MSP ID for the issuer wallet
+	IssuerMSPID = "IssuerMSPID"
+	// AuditorMSPID is the default MSP ID for the auditor wallet
+	AuditorMSPID = "AuditorMSPID"
+	// CertifierMSPID is the default MSP ID for the certifier wallet
+	CertifierMSPID = "CertifierMSPID"
+)
+
+// RoleToMSPID maps the role to the MSP ID
+var RoleToMSPID = map[driver.IdentityRole]string{
```

Comment:
this is a global variable. How do you plan to support multi-tenancy?

Reply:
But I thought you have a mapping from role to MSP ID? What if the MSP IDs are different across channels?

Reply:
but here is the point, these are not Fabric identities. Their management is different.